recipes/limereport/all/conanfile.py

```diff
@@ -0,0 +1,95 @@
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps
+from conan.tools.files import copy, get
+from conan import ConanFile
+from conan.tools.layout import basic_layout
+from conan.tools.microsoft import is_msvc
+from conan.tools.scm import Version
+from conan.tools.apple import fix_apple_shared_install_name
+from conan.errors import ConanInvalidConfiguration
+import os
+
+
+class LimereportConan(ConanFile):
+    name = "limereport"
+    description = "Report generator for Qt Framework"
+    homepage = "https://poppler.freedesktop.org/"
+    topics = ("limereport", "pdf", "report","qt")
+    license = "LGPL-3.0", "GPL-3.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_zint": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_zint": False,
+    }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "Visual Studio": "15",
+            "gcc": "8",
+            "clang": "8",
+            "apple-clang": "8"
+        }
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def build_requirements(self):
+        self.requires("libpng/[>=1.6 <1.7]")
+        if self.options.with_zint:
+            self.build_requires("zint/2.10.0")
+
+    def requirements(self):
+        self.requires("qt/6.4.2")
+
+    def validate(self):
+        if Version(self.dependencies["qt"].ref.version) < "6.0.0":
```

Comment:
Is this C++ 11 or C++ 17?

If the lib needs something greater than 11, CCI policy is to add a check in the validate method

As per https://github.com/fralx/LimeReport/blob/master/CMakeLists.txt#L50, you should be validating the cppstd version, but maybe Im missing something!



(see the template here https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/cmake_package/all/conanfile.py#L82 for how to do so in case it's necessary)

Reply:
Done